EarlGrey/Event/GREYSyntheticEvents.m

```diff
@@ -112,6 +113,41 @@ + (BOOL)rotateDeviceToOrientation:(UIDeviceOrientation)deviceOrientation
   return YES;
 }
 
++ (BOOL)shakeDeviceWithError:(NSError *__strong *)errorOrNil {
+  GREYFatalAssertMainThread();
+  
+  NSError *error;
+  BOOL success = [[GREYUIThreadExecutor sharedInstance] executeSyncWithTimeout:kRotationTimeout
+                                                                         block:^{
+    // Keep previous accelerometer events enabled value and force it to YES so that the shake
+    // motion is passed to the application.
+    UIApplication *application = [UIApplication sharedApplication];
```

Comment:
please use `application` everywhere in place of `[UIApplication sharedAppication]`